client/signup/config/steps-pure.js

```diff
@@ -734,6 +734,9 @@ export function generateSteps( {
 				'displayAddress',
 			],
 		},
+		'website-content': {
+			stepName: 'website-content',
```

Comment:
I think `siteSlug` should be a dependency since we'll be passing it via query params. Similar to: https://github.com/Automattic/wp-calypso/blob/14b5ad745c12fcab75168f9d7caa2ecb2456cadf/client/signup/config/steps-pure.js#L660-L667

Reply:
Will pick this up in the final integration.